### DISCUSSION TITLE
Bibliografy item numbered as zero, and input error - Overleaf

### DISCUSSION THREAD

**Discussion by sibguerra** [joaomlourenco/novathesis#313] [Q&A] [answered]
Hello!  

Why do we get zero reference number in some bibliografy items? How can I fix that?

![biblio6](https://user-images.githubusercontent.com/100953769/157066313-ef06a1da-3a0a-49f4-87f8-d6194168dd39.jpg)

And I have the following error:
![biblio7](https://user-images.githubusercontent.com/100953769/157066265-a37aa7fc-b65e-4ceb-9921-7dbf424168da.jpg)

**Comment by joaomlourenco** [comment, marked as answer]
At the right of the "compile" button there is a small arrow.  Click on it and select the last item "Recompile from scratch".  Hopefully it will solve the problem. If it does not, let me know… we will try the next step.  :)

**Reply by sibguerra** [reply]
Thank you João, I also found an epsilon character in one of the biblio references... 😒 After correcting to $\epsilon$, and "recompiling from scratch", the bibliografy turned out to be good. I did the same in TeXStudio, with the difference that, before compiling, I deleted all template files, except template.tex. I assumed that that was the "recompile from scratch" outside overleaf!

**Reply by sibguerra** [reply]
But should always recompile from scratch, every time I need to compile the text? It takes longer... I think that I should do that only when I get errors that are accumulating somewhere, I am right? Thank you!

**Reply by joaomlourenco** [reply]
Nope… recompile from scratch if and only there is something weird that can't be explained except by the forces of hell trying to boycott your work.  😉

**Reply by sibguerra** [reply]
... perfectly understood! 😁
